test/ConfigTest.groovy

```diff
@@ -0,0 +1,22 @@
+import static org.junit.Assert.*
+import org.junit.*
+
+class ConfigTest {
+  @Test
+  public void defaultRuleSet() {
+    def config = new Config([configFile: "/usr/src/app/fixtures/config.json", codeFolder: "/usr/src/app/fixtures"])
+    assertEquals config.ruleSet(), "/usr/src/app/ruleset.xml"
+  }
+
+  @Test
+  public void specifiedRuleSetFile() {
+    def config = new Config([configFile: "/usr/src/app/fixtures/config.rule.json", codeFolder: "/usr/src/app/fixtures"])
+    assertEquals config.ruleSet(), "/usr/src/app/fixtures/specified_rules.xml"
+  }
+
+  @Test
+  public void honorPresentRules() {
+    def config = new Config([configFile: "/usr/src/app/fixtures/config.json", codeFolder: "/usr/src/app/fixtures/honor"])
```

Comment:
What does the name `/honor` mean in the path? Is this a fixture directory just for this test?

If yes, I might suggest something more explicit like `/ruleset_default`, and maybe moving `specified_rules.xml` and `config.rule.json` to something like `/ruleset_specified`.

Reply:
@toddmazierski Re-structured `fixture` directory. Do you mind giving another shot?